docs/kb/rules/changing-rule-severity-and-other-metadata.md

```diff
@@ -0,0 +1,43 @@
+---
+description: Change rule severity and other metadata by forking rules
+tags:
+  - Rules
+  - Semgrep Registry
+  - Semgrep Code
+  - Semgrep Secrets
+append_help_link: true
+---
+
+# Change rule severity and other metadata by forking rules
+
+To alter the severity or other metadata of a Semgrep rule, it will first need to be forked and then updated. Forking means to copy or duplicate the rule, thereby creating your own custom version of it. Once this custom version is created, it can then be modified as needed.
```

Comment:
```suggestion
To alter the severity or other metadata of a Semgrep rule, it must be forked and then updated. Forking means to copy or duplicate the rule, thereby creating your own custom version of it. Once this custom version is created, it can then be modified as needed.
```

Reply:
```suggestion
To alter the severity or other metadata of a Semgrep rule, it myst be forked and then updated. Forking means to copy or duplicate the rule, creating your own custom version of it. Once you create the custom version, you can then modify the rule to suit your needs.
```